src/Layouts/Partials/AlignItemsPanel.vala

```diff
@@ -25,22 +25,36 @@ public class Akira.Layouts.Partials.AlignItemsPanel : Gtk.Grid {
 
     private struct AlignBoxItem {
         public string type;
-        public string action_name;
-        public string icon_name;
-        public string tooltip_text;
+        public string? action_name;
+        public string? icon_name;
+        public string? tooltip_text;
+        public string[]? accels;
+
+        public AlignBoxItem (
```

Comment:
We're not doing a hard wrap at any character length (maybe we should) so this indentation is pretty weird.

Reply:
I've done that because I think that it improves the readability of the code and helps to immediately tell what's the code is actually doing without scrolling the editor (I usually have more than one vertical split and so I need my rows to be consistent and short in size). It is also easier to add, delete or move attributes position inside the function call. In my opinion code is more organized this way. Regarding the act that we don't have a "max character length" rule, I think we should. I'd prefer 80, but also 120 (as recommended by Google C++ guidelines) might work as well. What do you think I should do with this code? :)

Reply:
Yeah, on second thought, we can leave it like this.
We will soon implement a character limit, probably a soft 80 and hard 100.
Thanks